src/server/app.yaml

```diff
@@ -17,3 +17,5 @@ resources:
 env_variables:
     GCLOUD_PROJECT: gator-01
 
+skip_files:
+    - ^tests$
```

Comment:
As written in https://cloud.google.com/appengine/docs/python/config/appref#skip_files the default exclusions list must be kept to extends it. We must keep it to exclude .pyc, .pyo, files ending with ~ ...